app/v1/models/errors/mtdErrors.scala

```diff
@@ -53,7 +56,9 @@ object TaxYearFormatError
 
 // Rule Errors
 object RuleTaxYearNotSupportedError
-    extends MtdError("RULE_TAX_YEAR_NOT_SUPPORTED", "Tax year not supported, because it precedes the earliest allowable tax year")
+    extends MtdError(
+      "RULE_TAX_YEAR_NOT_SUPPORTED",
+      "The specified tax year is not supported. That is, the tax year specified is before the minimum tax year value")
```

Comment:
This is the error message for the delete endpoint, but the other two (create/amend & retrieve) have the message as `The tax year specified does not lie within the supported range`

Which is better to go with? Should they all not be the same?